cmd/goQuery/commands/root.go

```diff
@@ -60,11 +64,10 @@ func init() {
 	subRootCmd.InitDefaultHelpCmd()
 	subRootCmd.InitDefaultHelpFlag()
 
-	rootCmd.Flags().BoolVarP(&cmdLineParams.External, "external", "x", false, helpMap["External"])
```

Comment:
Deprecated this. The handling of this special case is so awkward and non-standard.